### PR TITLE
fix/add img size mobile

### PR DESCRIPTION
**Pull request by fedgut**
## Quick Info
 Display diferent ad img src for mobile and for tablet and up

## What does this change?

- Fix seeds
- Add small_image_url to Ad
- Allow a litle space between posts
- Use diferent image sizes for ad
- Change breakpoints for mobile first

## Why are you changing that?
 For better ad usage

## Migrations?
Yes

## How do you manually test this?
open the rails console, an do  
```
ad = Ad.first
ad.mobile_img_url = 'https://www.magmalabs.io/packs/media/images/magma-logo-e56e4fa7dc892ffb0d967906890b3ecd.svg'
ad.save
```

then run `rails s`, press f12, and chose different viewports

### CODE REVIEW

File: db/migrate/20210224230319_add_smal_image_url_to_ads.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddSmalImageUrlToAds < ActiveRecord::Migration[5.0]
+  def change
+    add_column :ads, :small_image_url, :string, null: false
```
Comment (softr8) [inline db/migrate/20210224230319_add_smal_image_url_to_ads.rb]: what about `mobile_image_url` ? small not necessarily means it will be used in the mobile version of the site

File: db/seeds/development.rb [hunk 2]
```diff
@@ -3,6 +3,8 @@
 # To download production data instead of these seeds, run
 # rake db:restore_production_dump
 
+require 'faker'
```
Comment (softr8) [inline db/seeds/development.rb]: no need to load it here again

File: db/migrate/20210224230319_add_smal_image_url_to_ads.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 class AddSmalImageUrlToAds < ActiveRecord::Migration[5.0]
   def change
-    add_column :ads, :small_image_url, :string, null: false
+    add_column :ads, :small_image_url, :string, default: '', null: false
```
Comment (softr8) [inline db/migrate/20210224230319_add_smal_image_url_to_ads.rb]: you could've done: `git commit --fixup` to amend the original commit with the migration, as long as it is not merged into main master, we can do that

File: db/seeds/development.rb (hunk 2 above)
Comment (fedgut) [reply db/seeds/development.rb]: True! will remove

File: db/migrate/20210224230319_add_smal_image_url_to_ads.rb (hunk 1 above)
Comment (fedgut) [reply db/migrate/20210224230319_add_smal_image_url_to_ads.rb]: that's a good point! I'll, change the naming

File: db/migrate/20210224230319_add_smal_image_url_to_ads.rb (hunk 3 above)
Comment (fedgut) [reply db/migrate/20210224230319_add_smal_image_url_to_ads.rb]: That's a good point! I will rebase to squash some commit

File: app.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (softr8) [inline app.json]: make this change in a separate PR, right now, the review apps are using production db just to have real data, not sure if we need to tests super often new posts

File: db/seeds.rb [hunk 5]
```diff
@@ -1,12 +1,14 @@
 # frozen_string_literal: true
 
-Dir[Rails.root.join('db/seeds/common/*.rb')].sort.each do |file|
-  puts "Loading file #{file.split(/\/./).last}"
+require 'faker'
+
+Dir[Rails.root.join('db/seeds/common/*.rb')].each do |file|
+  puts "\n Loading file #{file.split(/\//).last}"
```
Comment (softr8) [inline db/seeds.rb]: These changes do not seem to belong to fixing mobile ads images, no big deal, but just need to have more concise PR's

File: app.json (hunk 4 above)
Comment (fedgut) [reply app.json]: totally will! Ill revert that change

File: db/seeds.rb (hunk 5 above)
Comment (fedgut) [reply db/seeds.rb]: Thats true! Ill keep the PR single purpose in the future!

File: app/views/posts/_post.html.haml [hunk 6]
```diff
@@ -36,4 +36,5 @@
 - if post.id.to_i.odd? || action_name == 'show'
   - if ad = Ad.random
     .magma_container
-      = link_to(image_tag(ad.image_url), clicks_path(id: ad), method: :post)
\ No newline at end of file
+      = link_to(image_tag(ad.image_url), clicks_path(id: ad), method: :post)
```
Comment (softr8) [inline app/views/posts/_post.html.haml]: weren't you able to use srcset to use images per window resolution?
```haml
= link_to image(tag.image_url, srcset: "#{tag.small_image_url} 480w", sizes: "(max-width: 480px) 440px", clicks_path(ad), method: :post)
```

**Review by fabiannabi — APPROVED** [review]
LGTM~
